tests/system_tests/test_plans.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+import pytest
+from blueapi.client.client import BlueapiClient
+from blueapi.client.event_bus import AnyEvent
+from blueapi.core.bluesky_types import DataEvent
+from blueapi.worker.event import TaskStatus, WorkerEvent, WorkerState
+from blueapi.worker.task import Task
+
+_DISABLE_SIDE_EFFECTS = bool(os.environ.get("DISABLE_SIDE_EFFECTS", 0))
+_DISABLE_SIDE_EFFECTS_MESSAGE = """
+    This test would cause side effects on the beamline, it has been disabled
+    so as not to interfere with operation. To run tests that may interfere with
+    the beamline export DISABLE_SIDE_EFFECTS=0 or add to add to env in pyproject.toml
+    """
+disable_side_effects = pytest.mark.skipif(
+    _DISABLE_SIDE_EFFECTS, reason=_DISABLE_SIDE_EFFECTS_MESSAGE
+)
```

Comment:
Could: I know I wrote this but I'm on the fence about keeping it for a few reasons:

1. It relies on people who write tests to remember to invoke it, so it's not that reliable anyway
2. The expectation is that these tests will be run manually by someone who knows what they are doing
3. The paradigm for system tests has changed since I wrote this, we are probably going to run these at set times when we have booked the beamline/test rig
4. It would be simpler (and not particularly problematic) to not have them and just run all the tests every time

Let me know your thoughts